templates/zerver/emails/invitation.source.html

```diff
@@ -5,22 +5,26 @@
 {% endblock %}
 
 {% block content %}
-<p>Hi there,
+<p>{{ _("Hi there,") }}
 </p>
 <p>
-    {{ referrer.full_name }} ({{ referrer.email }}) wants you to join
+    {% trans %}
+    {{ referrer_full_name }} ({{ referrer_email }}) wants you to join
     them on Zulip &mdash; the team communication tool designed for
     productivity.
+    {% endtrans %}
 </p>
 <p>
-    To get started, please click here:
-    <a class="button" href="{{ activate_url }}">Complete registration</a>
+    {{ _("To get started, please click here:") }}
+    <a class="button" href="{{ activate_url }}">{{ _("Complete registration") }}</a>
 </p>
 <p>
-    Feel free to give us a shout at <a href="mailto:{{ support_email }}">{{ support_email }}</a>, if you have any questions.
+    {% trans %}Feel free to give us a shout at <a href="mailto:{{ support_email }}">{{ support_email }}</a>, if you have any questions.{% endtrans %}
```

Comment:
For this string, which only differs between the plain-text and full-email versions here, I could imagine having a template variable `support_email_link` that is passed in through the context, and then this line would be identical to the `.txt` line....

Reply:
(thinking about this more generally, though, it's possible the right long-term strategy here is to figure out how to generate the plain-text versions from the HTML versions)

Reply:
(For context, this is what this looks like for translators)

![image](https://user-images.githubusercontent.com/2746074/39380808-b52c0738-4a14-11e8-90b2-1e5e2e8e83c4.png)